k8s/anthos/machines.go

```diff
@@ -0,0 +1,76 @@
+package anthos
+
+import (
+	"context"
+	"encoding/json"
+
+	v1alpha1 "sigs.k8s.io/cluster-api/pkg/apis/deprecated/v1alpha1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	machineResource = schema.GroupVersionResource{Group: "cluster.k8s.io", Version: "v1alpha1", Resource: "machines"}
+)
+
+// MachinesOps is an interface to perform k8s machines operations
+type MachineOps interface {
+	// ListMachines lists all machines in kubernetes cluster
+	ListMachine(ctx context.Context) (*v1alpha1.MachineList, error)
+	// GetMachine returns a machine for the given name
+	GetMachine(ctx context.Context, name string) (*v1alpha1.Machine, error)
+	// DeleteMachine delete machine for given name
+	DeleteMachine(ctx context.Context, name string) error
+}
+
+// ListMachines lists all machines in kubernetes cluster
+func (c *Client) ListMachine(ctx context.Context) (*v1alpha1.MachineList, error) {
```

Comment:
ListMachines would be a better name for the function and will match your comment on top

Reply:
Fixed it..